reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -186,6 +190,28 @@ public Emission getReason() {
 		}
 	}
 
+	/**
+	 * A handler for any non-successful emission result from operations
+	 * such as {@link Many#emitNext(Object, EmitFailureHandler)}
+	 * that allows retrying or failing the operation.
+	 */
+	public interface EmitFailureHandler {
+
+		/**
+		 * A pre-made handler that will not attempt at retry any failure
```

Comment:
change beginning of sentence: "will not ~attempt at~ **instruct to** retry", add `,` at the end of the line, replace `trigger` below with `triggering`.